spec/dumped/repositories/repo_repository_spec.rb

```diff
@@ -1,15 +1,15 @@
 require_relative '../../spec_helper'
 
-describe GitRepoRepository do
-  let(:repo) { GitRepoRepository.new }
+describe RepoRepository do
+  let(:repo) { RepoRepository.new }
 
   after { repo.clear }
 
   describe '#by_language' do
     before do
-      repo.create(git_repo_attributes(language: 'ruby'))
-      repo.create(git_repo_attributes(language: 'ruby'))
-      repo.create(git_repo_attributes(language: 'javascript'))
+      repo.create(repo_attributes(language: 'ruby'))
+      repo.create(repo_attributes(language: 'ruby'))
+      repo.create(repo_attributes(language: 'javascript'))
```

Comment:
try to use fabricator instead, I hope it will be much easier than now
https://github.com/jodosha/hanami-fabrication